target/linux/realtek/dts/rtl8380_panasonic_m8eg-pn28080k.dts

```diff
@@ -0,0 +1,280 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl838x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "panasonic,m8eg-pn28080k", "realtek,rtl838x-soc";
+	model = "Panasonic Switch-M8eG PN28080K";
+
+	aliases {
+		led-boot = &led_status_green;
+		led-failsafe = &led_status_amber;
+		led-running = &led_status_green;
+		led-upgrade = &led_status_green;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,9600";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x8000000>;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		collision {
+			label = "amber:any_col";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_amber: status_amber {
+			label = "amber:status_eco";
+			gpios = <&gpio2 1 GPIO_ACTIVE_LOW>;
+		};
+
+		led_status_green: status_green {
+			label = "green:status_eco";
+			gpios = <&gpio2 2 GPIO_ACTIVE_LOW>;
+		};
+
+		gigabit {
+			label = "green:giga";
+			gpios = <&gpio2 8 GPIO_ACTIVE_LOW>;
+		};
+
+		100mbps {
+			label = "green:100m";
+			gpios = <&gpio2 9 GPIO_ACTIVE_LOW>;
+		};
+
+		full_duplex {
+			label = "green:full";
+			gpios = <&gpio2 10 GPIO_ACTIVE_LOW>;
+		};
+
+		loop_history {
+			label = "green:loop_history";
+			gpios = <&gpio2 11 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
As always, I'm not really a fan of mixing different "names" for node and label here. If it's full duplex, why not call the led "green:full-duplex" as well so the name will actually explain what it's for?

Why is the label status_eco and the node just "status". And what's any_col? ...

Reply:
@adschm I used the generic name for node name, and for label, I used the name that is actually printed.

![IMG_20210529_103801](https://user-images.githubusercontent.com/10043798/120054309-208b5280-c06a-11eb-85e7-800d7d45617c.jpg)

> Why is the label status_eco and the node just "status"

It's my bad, I'll change the node name to ```status_eco```.

> what's any_col

If this mode is selected, the port LED where the collision has occurred so far will be turned on.

Reply:
Okay, then just using "giga" and "100m" for the node names would be most consistent.

Reply:
Thank you, I'll update the names.